assets/app/styles/_sidebar.less

```diff
@@ -192,8 +191,14 @@
               }
               ul.nav-sidenav-secondary {
                 margin-bottom: 20px;
-                > li > a {
-                  padding: 10px 0;
+                > li {
+                  &.active a {
```

Comment:
Doesn't look like the immediate child selector is necessary, at least not now. Not sure why it was there before.